tests/hpc/rasdaemon.pm

```diff
@@ -0,0 +1,40 @@
+# SUSE's openQA tests
+#
+# Copyright © 2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: rasdaemon may have to be tested on a physical system, as EDAC support
+#may not be available in a VM. To check this, install rasdaemon:
+# zypper in rasdaemon and run
+# ras-mc-ctl --status.
+#If the message returned indicates that there is no EDAC driver loaded, the system  does
+# not support it thus not all tests below can be performed
+# Maintainer: Anton Smorodskyi <asmorodskyi@suse.com>
+# Tags: https://fate.suse.com/318824
+
+use base 'hpcbase';
+use strict;
+use testapi;
+use utils;
+
+sub run {
+    my $self = shift;
+
+    # Install genders package
+    zypper_call('in rasdaemon');
+
+    script_run('ras-mc-ctl --status');
```

Comment:
You say it's failing but I do not see this in the test run. Shouldn't the border show up as red? In other case, you could run `assert_script_run('! ras-mc-ctl --status', fail_message => 'it should fail because <here is where @asmorodskyi enters more details>');`

Reply:
```
@okurz yes, it is failing when you run (it exits with "1" as return code)
sudo ras-mc-ctl --status
ras-mc-ctl: drivers not loaded.
echo $?
1
```
@asmorodskyi  I rechecked fate and played with rasdaemon and i believe that status command should run after is rasdaemon started. Fate can be little bit unclear, but it doesn't make sense at the moment.